.gitignore

```diff
@@ -2,3 +2,4 @@ bin/
 demo/helper-scripts/*.pdf
 demo/helper-scripts/*.log
 /kustomization.yaml
+.idea
```

Comment:
Yes true. However, I thought that it would be great to add it for users who use JetBrains products. I will open another PR for it if you see that it is helpful. WDYT?